### ISSUE TITLE
Angular error: [$sanitize:badparse] The sanitizer was unable to parse the following.

### ISSUE THREAD

**Issue by nickstenning** [hypothesis/h#2935]
### Steps to reproduce

1. Go to http://www.nytimes.com/2015/01/17/science/earth/2014-was-hottest-year-on-record-surpassing-2010.html and activate Hypothesis.

### Result

The console contains this error:

```
angular.min.js?d1322f9d:5 Error: [$sanitize:badparse] The sanitizer was unable to parse the following block of html: <img</a> src="http://ipccar5summary.org/wp-content/uploads/2014/10/WGI_FSPM1b_Observed-change-in-surface-temperature-1901%E2%80%932012.png" alt="Image Description" /></p>
http://errors.angularjs.org/1.4.7/$sanitize/badparse?p0=%3Cimg%3C%2Fa%3E%20…%2580%25932012.png%22%20alt%3D%22Image%20Description%22%20%2F%3E%3C%2Fp%3E
    at angular.min.js?d1322f9d:1
    at htmlParser (angular-sanitize.min.js?ea3db86c:1)
    at angular-sanitize.min.js?ea3db86c:1
    at app.min.js?63e82ed1:2
    at renderMathAndMarkdown (app.min.js?63e82ed1:2)
    at ctrl.$render (app.min.js?63e82ed1:2)
    at Object.ngModelWatch (angular.min.js?d1322f9d:8)
    at Scope.$digest (angular.min.js?d1322f9d:6)
    at app.min.js?63e82ed1:3
    at render (app.min.js?63e82ed1:3)
```

**Comment by robertknight** [comment]
See also #2970

**Comment by nickstenning** [comment]
According to @robertknight this was fixed by upgrading to Angular 1.5. This bug no longer repros on the page linked.
